runestone/activecode/js/activecode.js

```diff
@@ -1485,33 +1475,21 @@ ACFactory.createActiveCode = function (orig, lang, addopts) {
 }
 
 // used by web2py controller(s)
-ACFactory.addActiveCodeToDiv = function(outerdivid, acdivid, sid, initialcode, language) {
+ACFactory.addActiveCodeToDiv = function(outerdiv, acdiv, sid, initialcode, language) {
     var  thepre, newac;
-
```

Comment:
this group of changes was made to work around a problem with jquery and usernames that contains . or other special jquery characters.